### DISCUSSION TITLE
An username already taken but I want that

### DISCUSSION THREAD

**Discussion by notcodev** [community/community#142556] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone! I wanted to create organization with @voidofcode nickname but it is already taken and account is empty, no email or something different to contact account owner, how can I contact to owner of this account?

**Comment by tylerebowers** [comment]
Some users have an email on their profile. You also my be able to check their commits to see if they have an email in one of them. Github removed PMs years ago unfortunately. I don't think that any of these would work, your best bet is pick a new nickname.

**Comment by FirstTimeEZ** [comment]


**Reply by tylerebowers** [reply]
It has one bit of activity in the recent year. Probably counts as active.

**Reply by notcodev** [reply]
> It has one bit of activity in the recent year. Probably counts as active.

This activity is "Joined GitHub", account has not activity except this one, but maybe account has a private repos and other activity doesn't shows

**Reply by notcodev** [reply]
Anyway, thank you for the answer, I'll try to contact support

**Reply by notcodev** [reply]
Unfortunately, GitHub support no longer accepts requests to release usernames😢
I will hope that owner of this account will see that topic in his mail and contact me to discuss about transferring username.

**Comment by thomassiard** [comment]
Hello!

I understand your frustration with trying to create an organization with the @VoidOfCode nickname when it’s already taken. Unfortunately, if the account is empty and there’s no contact information available, it can be challenging to reach out to the owner directly. Here are a few steps you might consider:

Check the Profile:

Visit the profile of @VoidOfCode on GitHub to see if there are any details provided, like a website or social media links, that might help you contact the owner.
GitHub Support:

You can contact [GitHub Support](https://support.github.com/contact) and explain the situation. While they typically cannot disclose user information, they may have procedures in place to assist in such cases.
Username Availability:

If you’re unable to contact the owner, keep an eye on the account. If it becomes inactive for a significant period, GitHub may release the username back into the pool of available usernames.
Alternative Names:

In the meantime, consider using a variation of the name you wanted, perhaps by adding a prefix or suffix, to create a unique organization name.
I hope this helps you find a way forward! Good luck!

**Reply by notcodev** [reply]
Thank you for the answer!

**Reply by ebndev** [reply]
Hi @notcodev,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
